exts/cesium.omniverse/cesium/omniverse/ui/statistics_widget.py

```diff
@@ -46,7 +52,8 @@ def _on_update_frame(self, _e: carb.events.IEvent):
         if not self.visible:
             return
 
-        fabric_statistics = self._cesium_omniverse_interface.get_fabric_statistics()
+        render_statistics = self._cesium_omniverse_interface.get_render_statistics()
+        fabric_statistics = render_statistics.fabric_statistics
```

Comment:
NIT: I have mixed feelings here. There's little reason to pull the `fabric_statistics` property out here into it's own variable. It doesn't help readability that much and I think an argument can be made that we're hiding that the fabric statistics are part of render_statistics.

On the other hand... :shrug:

Reply:
I was on the fence about this too. I was trying to simplify the C++ side but I think a flat struct is cleaner.